src/js/components/form-elements/FilterSelectField.jsx

```diff
@@ -66,7 +74,10 @@ const Menu = (props) => {
       target={inputContainer}
       container={document.getElementById('root')}
     >
-      <Dropdown inputContainerRec={inputContainer.getBoundingClientRect()} >
+      <Dropdown
+        inputContainerRec={inputContainer.getBoundingClientRect()}
+        hasOptions={!!props.options.length}
```

Comment:
Are `props.options` required? Is it possible it will be null? Should it be a part of `propTypes`?

Reply:
`options` prop from the react-select is an empty array by default.